2019-01-07-fedora-29-template-available.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "Fedora 29 TemplateVM available for Qubes 4.0"
+date: 2019-01-07
+categories: announcements
+author: Andrew David Wong
+---
+
+A new Fedora 29 TemplateVM is now available for Qubes 4.0.  We
+previously announced that [Fedora 27 reached EOL] and encouraged users
+to upgrade to Fedora 28. Fedora 28 is still supported by the Fedora
+Project, so users may now choose either Fedora 28 or 29 (or both)
+depending on their needs and preferences.  Instructions are available
+for [upgrading from Fedora 28 to 29].  We also provide fresh Fedora 29
+TemplateVM packages through the official Qubes repositories, which you
+can get with the following commands (in dom0).
+
+Standard Fedora 29 TemplateVM:
+
+    $ sudo qubes-dom0-update qubes-template-fedora-29
+
+[Minimal] Fedora 29 TemplateVM:
+
+    $ sudo qubes-dom0-update qubes-template-fedora-29-minimal
+
+After upgrading to a Fedora 29 TemplateVM, please remember to set all
+qubes that were using the old template to use the new one. This can be
+done in dom0 either with the Qubes VM Manager or with the `qvm-prefs`
+command-line tool.
```

Comment:
Since not that far ago, there is new tool on Qubes 4.0 for that: https://github.com/QubesOS/qubes-issues/issues/4085
I've updated [doc](https://www.qubes-os.org/doc/templates/#how-to-switch-templates-40) already (but without up to date screenshot). Maybe it worth linking to that doc section?

Reply:
Good idea.